spec/models/manageiq/providers/ansible_tower/configuration_manager/configuration_script/api_create_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'ansible_tower_client'
+
+describe ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationScript do
+  context "::ApiCreate" do
+    let(:provider)      { FactoryGirl.create(:provider_ansible_tower, :with_authentication) }
+    let(:manager)       { provider.managers.first }
+    let(:atc)           { double("AnsibleTowerClient::Connection", :api => api) }
+    let(:api)           { double("AnsibleTowerClient::Api", :job_templates => job_templates) }
+    let(:job_templates) { double("AnsibleTowerClient::Collection", :create => job_template) }
+    let(:job_template)  { AnsibleTowerClient::JobTemplate.new(nil, params) }
+    let(:params) do
+      {
+        :description  => "Description",
+        :extra_vars   => {}.to_json,
+        :id           => 1,
+        :inventory_id => 1,
```

Comment:
At job template creation time user may already select hosts, although at launching time they could be updated. How can we add hosts at job template creation time?

Also I assume the credentials can be part of `params`, right?